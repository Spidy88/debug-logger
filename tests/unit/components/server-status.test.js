import React from 'react';
import renderer from 'react-test-renderer';
import { render } from 'enzyme';
import ServerStatus from '../../../client/components/server-status';

describe('<ServerStatus>', () => {
    it('should render the provided status message', () => {
        const statusMessage = 'Testing in progress';
        const serverStatus = render(<ServerStatus status={statusMessage} />);

        expect(serverStatus.text()).toEqual(statusMessage);
    });

    it('should match its snapshot', () => {
        const statusMessage = 'Testing in progress';
        const tree = renderer.create(<ServerStatus status={statusMessage} />);

        expect(tree.toJSON()).toMatchSnapshot();
    });
});