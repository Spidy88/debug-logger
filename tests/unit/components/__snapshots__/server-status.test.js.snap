// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServerStatus> should match its snapshot 1`] = `
<div
  className="ðŸ“¦bg_1whhwgh ðŸ“¦pb_1rem ðŸ“¦pt_1rem ðŸ“¦dspl_flex ðŸ“¦algn-itms_center ðŸ“¦just-cnt_center ðŸ“¦box-szg_border-box css-nil"
>
  <span
    className="ðŸ“¦color_425A70 ðŸ“¦fnt-fam_b77syt ðŸ“¦fnt-sze_14px ðŸ“¦f-wght_700 ðŸ“¦ln-ht_20px ðŸ“¦ltr-spc_-0-05px ðŸ“¦box-szg_border-box"
  >
    Testing in progress
  </span>
</div>
`;
